lib/internal/test_runner/reporter/dot.js

```diff
@@ -1,15 +1,20 @@
 'use strict';
+
+const { green, red, clear } = require('internal/util/colors');
 const { MathMax } = primordials;
 
 module.exports = async function* dot(source) {
   let count = 0;
   let columns = getLineLength();
+  const shouldColorize = process.stdout.isTTY || process.env.FORCE_COLOR !== undefined;
```

Comment:
@bantoinese83 this is redundant, as the `green`, `red`, and `clear` values already handle this 🙂